aries-site/src/pages/components/checkbox.js

```diff
@@ -24,11 +33,146 @@ const CheckBox = () => (
         </SubsectionText>
         <Example
           docs="https://v2.grommet.io/checkbox?theme=hpe#props"
-          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/CheckboxExample.js"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/checkbox/CheckboxSimpleExample.js"
+          figma="https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1744%3A1"
+          designer="https://designer.grommet.io/home?id=HPE-design-system-taylor-seamans-hpe-com"
+        >
+          <CheckBoxSimpleExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBox can be used for a list of different options that are being
+          provided to a user to select a option as well as being able to select
+          multiple options.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox can be used in a form of toggle. A single option which can
+          indicate something such as on or off. Toggle is prefered to quickly
+          switch between two possible states. Toggles usually provide immediate
+          results in which gives users the ability to control their preference.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox should not be used if a user can only choose one option from
```

Comment:
CheckBox should not be used if a user should only be allowed to select one option from a list of options. In the case that a user should only be able to select one option, refer to{' '}